### ISSUE TITLE
시스템6 성능 실험

### ISSUE THREAD

**Issue by februa22** [nlp-research/biaffineparser#42]

**Comment by februa22** [comment]
특징:

- Pre-trained 단어 embedding (200-dim)
- BiLSTM으로 단어 기반 어절 벡터 표현 생성
- Pre-trained POS태그 임베딩 (100-dim)
- BiLSTM으로 POS태그 기반 어절 벡터 표현 생성
- 단어 기반 어절 벡터 + POS태그 기반 어절 벡터

**Comment by februa22** [comment]
Hyperparameters:

```json
{
    "num_lstm_units": 400,
    "num_lstm_layers": 3,
    "arc_mlp_units": 500,
    "label_mlp_units": 100,
    "word_embed_size": 200,
    "pos_embed_size": 100,
    "optimizer": "adam",
    "learning_rate": 0.001,
    "decay_factor": 0.9,
    "num_train_epochs": 100,
    "train_filename": "data/sejong.train.csv",
    "dev_filename": "data/sejong.test.csv",
    "out_dir": "parser/model_iss42",
    "word_vocab_name": "word.pkl",
    "pos_vocab_name": "pos.pkl",
    "rel_vocab_name": "rel.pkl",
    "head_vocab_name": "head.pkl",
    "word_embed_file": "embeddings/words.pos.original.vec",
    "pos_embed_file": "embeddings/words.tag.original.vec",
    "embed_dropout": 0.33,
    "lstm_dropout": 0.33,
    "mlp_dropout": 0.33,
    "batch_size": 128,
    "device": "gpu",
    "debug": false
}
```

**Comment by februa22** [comment]
성능:

> epoch: 12 - loss: 0.22962340712547302 - uas: 0.942307710647583 - las: 0.9254807829856873 - eval_loss: 0.6107974648475647 - eval_uas: 0.8865121006965637 - eval_las: 0.8526851534843445

**Comment by februa22** [comment]
성능: 

> epoch: 9 - loss: 0.20490193367004395 - uas: 0.937901496887207 - las: 0.9336188435554504 - eval_loss: 0.4963613450527191 - eval_uas: 0.8967400193214417 - eval_las: 0.865535318851471

**Comment by februa22** [comment]
불완전한 데이터셋으로 학습한 결과인 것 같아서, 재실험 진행 중.

**Comment by februa22** [comment]
데이터셋 교체 후 성능:

> epoch: 7 - loss: 0.24888640642166138 - uas: 0.9233193397521973 - las: 0.9075630307197571 - eval_loss: 0.39266616106033325 - eval_uas: 0.9161076545715332 - eval_las: 0.8915930390357971
